sentry-android-core/src/test/java/io/sentry/android/core/DefaultAndroidEventProcessorTest.kt

```diff
@@ -376,4 +377,16 @@ class DefaultAndroidEventProcessorTest {
 //            assertNotNull(device.connectionType)
         }
     }
+
+    @Test
+    fun `Event sets language and locale`() {
+        val sut = fixture.getSut(context)
+        val locale = Locale.getDefault()
+
+        assertNotNull(sut.process(SentryEvent(), null)) {
+            val device = it.contexts.device!!
+            assertEquals(device.language, locale.language)
```

Comment:
can we assert against raw string values? because we are using the same function (getDefault) as in production code, so if it changes, our test in theory should fail, but it won't.

I think I'd set the default to `Locale.setDefault(Locale.GERMAN)` for instance, then assert against "de_DE" and so on